src/phongo_client.c

```diff
@@ -75,7 +75,7 @@ static const mongoc_client_t* get_first_pclient_client(HashTable* ht)
  * Note: this may incorrectly return NULL if crypt_shared was loaded through a
  * mongoc_client_t since destroyed (e.g. single requested-scoped client);
  * however, that's the best can do with libmongoc's API. */
-const char* php_phongo_crypt_shared_version()
+const char* php_phongo_crypt_shared_version(void)
```

Comment:
Very interesting considering I addressed this exact issue in [PHPC-2166](https://jira.mongodb.org/browse/PHPC-2166) (and related libmongoc/libmongocrypt tickets). I'm surprised that this was overlooked. Please double-check that it's the last occurrence before merging. Also, feel free to reference PHPC-2166 in the commit message to link it up (we don't need to change the fixVersion).

Reply:
I checked, and there are none left in PHPC :)